opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
> but there is some overlap with the enabled message concept in the Logger class so perhaps we should refactor that instead?

As I see it there is 100% overlap with the `LogBackend::m_mask` field - the `LogBackEnd::m_mask` field configures exactly which message types will be included by a particular logging backend, and then this `level` makes the same decision one more time - with lesser flexibility.

If the `LogBackend::m_mask` functionality is deemed not to work we should fix that (or remove it); but not just add another configuration option. Exactly which problem does the `level` solve?